app/ide-desktop/lib/dashboard/src/components/AriaComponents/Text/Text.tsx

```diff
@@ -185,5 +186,5 @@ Text.Heading = React.forwardRef(function Heading(
   ref: React.Ref<HTMLHeadingElement>
 ) {
   const { level = 1, ...textProps } = props
-  return <Text ref={ref} elementType={`h${level}`} variant="h1" {...textProps} />
+  return <Text ref={ref} elementType={`h${level}`} variant="h1" balance {...textProps} />
```

Comment:
is `balance` necessary here?

Reply:
yes, because this makes headers look prettier https://developer.chrome.com/docs/css-ui/css-text-wrap-balance